### PR TITLE
Added Settings Hotkey

### PR DESCRIPTION
**Pull request by Jossos**
Added hotkey to open settings
Fixes [AB#1487](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/1487)

### CODE REVIEW

File: src/vcSettings.cpp [hunk 1]
```diff
@@ -359,6 +359,7 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     vcHotkey::Set(vcB_ToggleMeasureAreaTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleMeasureAreaTool)).AsInt(65));
     vcHotkey::Set(vcB_ToggleInspectionTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleInspectionTool)).AsInt(66));
     vcHotkey::Set(vcB_ToggleAnnotateTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleAnnotateTool)).AsInt(67));
+    vcHotkey::Set(vcB_OpenSettingsMenu, data.Get("keys.%s", vcHotkey::GetBindName(vcB_OpenSettingsMenu)).AsInt(68));
```
Comment (samsinsane) [inline src/vcSettings.cpp:362]: This shortcut won't work in the browser - F11 in Chrome will fullscreen the page.

File: src/main.cpp [hunk 2]
```diff
@@ -352,6 +352,11 @@ void vcMain_MainLoop(vcState *pProgramState)
   io.KeysDown[SDL_SCANCODE_BACKSPACE] = false;
   io.KeysDown[SDL_SCANCODE_PRINTSCREEN] = false;
 
+  if (vcHotkey::IsPressed(vcB_OpenSettingsMenu))
+  {
```
Comment (samsinsane) [inline src/main.cpp:356]: Just an FYI, braces aren't required here.

File: src/vcSettings.cpp [hunk 3]
```diff
@@ -359,6 +359,7 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     vcHotkey::Set(vcB_ToggleMeasureAreaTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleMeasureAreaTool)).AsInt(65));
     vcHotkey::Set(vcB_ToggleInspectionTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleInspectionTool)).AsInt(66));
     vcHotkey::Set(vcB_ToggleAnnotateTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleAnnotateTool)).AsInt(67));
+    vcHotkey::Set(vcB_OpenSettingsMenu, data.Get("keys.%s", vcHotkey::GetBindName(vcB_OpenSettingsMenu)).AsInt(vcMOD_Ctrl + 17)); // CTRL + N
```
Comment (samsinsane) [inline src/vcSettings.cpp:362]: Ctrl+N is the shortcut to open a new tab in Chrome.

Comment (Jossos) [reply src/vcSettings.cpp:362]: lol

Comment (Jossos) [reply src/vcSettings.cpp]: ok i made it ctrl M

Comment (samsinsane) [reply src/vcSettings.cpp]: Hate to say this, but Ctrl+M doesn't work on Linux - and I don't think it will work on macOS either.

Comment (Jossos) [reply src/vcSettings.cpp]: doesn't work?

**Pull request merged**